installation/install

```diff
@@ -85,11 +81,12 @@ function handle_error
 
 function get_os_type
 {
-	which zypper &> /dev/null && os="opensuse"
 	which pacman &> /dev/null && os="archlinux"
-	which dnf &> /dev/null && os="fedora"
 	which apt-get &> /dev/null && os="debian"
+	which dnf &> /dev/null && os="fedora"
 	which emerge &> /dev/null && os="gentoo"
+	which zypper &> /dev/null && os="opensuse"
```

Comment:
Oh nice, I understand & like that, let's keep it that way then! 

I think you should have made a commit which mentioned this, so there would be no misunderstanding